perf-cluster-operation/pkg/benchmark.go

```diff
@@ -124,7 +124,7 @@ func clusterConfigForBenchmark(benchmarkName, benchmarkRoot string) ClusterConfi
 
 // clusterNameForBenchmark prepends repo name to the benchmark name, and use it as the cluster name.
 func clusterNameForBenchmark(benchmarkName, repo string) string {
-	return fmt.Sprintf("%s-%s", repo, benchmarkName)
+	return fmt.Sprintf("%s--%s", repo, benchmarkName)
```

Comment:
I think this deserves a `repoPrefix(repo string) string` function

Reply:
Done.